dev/integration_tests/flutter_gallery/lib/demo/material/overscroll_demo.dart

```diff
@@ -27,8 +27,10 @@ class OverscrollDemoState extends State<OverscrollDemo> {
 
   Future<void> _handleRefresh() {
     final Completer<void> completer = Completer<void>();
-    Timer(const Duration(seconds: 3), () { completer.complete(); });
+    Timer(const Duration(seconds: 3), () => completer.complete());
     return completer.future.then((_) {
+      if (!mounted)
```

Comment:
There is a bug, trigger as following steps:
1. Open the `Pull to refresh` page
2. Pull out the `RefreshIndicator` by dragging
3. Page back before the `_handleRefresh` future complete.
4. Bug triggered. (use `context` after disposal assert)

Reply:
This seems to exist before the change?

Reply:
yes,  the bug exist before this change and 
this change can trigger the bug.